test/common/assertTransitConditions.js

```diff
@@ -0,0 +1,13 @@
+const expect = require('./expect');
+
+module.exports = actualDrugItem => async ({
+  from, to, whenHandoverLogIndex, expectedConditions,
+}) => {
+  const actualTransitConditions = await actualDrugItem.getTransitConditions(
+    from,
+    to,
+    (await actualDrugItem.handoverLog(whenHandoverLogIndex)).when,
+  );
+  expect(actualTransitConditions.temperature).to.equal(`${expectedConditions.temperature}`);
```

Comment:
I think this: `expect(+actualTransitConditions.temperature).to.equal(expectedConditions.temperature);` will also compare string and number

Reply:
Possibly yes but it feels more hacky although it's more concise.